src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -129,6 +130,53 @@ public void testGetTableData() throws SQLException {
         assertArrayEquals(expected, actual);
     }
 
+    @Test
+    public void testDataSetToString() {
```

Comment:
Testing of the method toString() from DataSet class should provide DataSetTest but not DatabaseManagerTest

Reply:
Yes, I agree with you. I will move this test to DataSetTest when we will be test this class